src/pages/Login.js

```diff
@@ -0,0 +1,17 @@
+const Auth_url = "https://accounts.spotify.com/authorize";
+const Redirect_url = "http://localhost:3000/playlists";
```

Comment:
does localhost 3000 need to be hardcoded? cant it be smthng like /playlists? ... so its more versatile.. if the root changes etc.?